tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -1447,7 +1481,7 @@ public function testMergeWithSingleAssociationAndFieldLists()
 
         $marshall = new Marshaller($this->articles);
         $marshall->merge($article, $data, [
-            'fieldList' => ['title', 'user'],
+            $fields => ['title', 'user'],
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?